<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QMainWindow" name="Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Калькулятор</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:rgb(218, 215, 255)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_e">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>290</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
font: 12pt &quot;Tw Cen MT Condensed Extra Bold&quot;;
background-color:rgb(194, 179, 243)
}

QPushButton:pressed
{
background-color:rgb(153, 147, 199);
}
</string>
    </property>
    <property name="text">
     <string>e</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
font: 12pt &quot;Tw Cen MT Condensed Extra Bold&quot;;
background-color:rgb(179, 174, 218)
}

QPushButton:pressed
{
background-color:rgb(153, 147, 199);
}
</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
font: 12pt &quot;Tw Cen MT Condensed Extra Bold&quot;;
background-color:rgb(179, 174, 218)
}

QPushButton:pressed
{
background-color:rgb(153, 147, 199);
}
</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
font: 12pt &quot;Tw Cen MT Condensed Extra Bold&quot;;
background-color:rgb(179, 174, 218)
}

QPushButton:pressed
{
background-color:rgb(153, 147, 199);
}
</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
font: 12pt &quot;Tw Cen MT Condensed Extra Bold&quot;;
background-color:rgb(179, 174, 218)
}

QPushButton:pressed
{
background-color:rgb(153, 147, 199);
}
</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>290</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
font: 12pt &quot;Tw Cen MT Condensed Extra Bold&quot;;
background-color:rgb(179, 174, 218)
}

QPushButton:pressed
{
background-color:rgb(153, 147, 199);
}
</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_point">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>350</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
font: 12pt &quot;Tw Cen MT Condensed Extra Bold&quot;;
background-color:rgb(194, 179, 243)
}

QPushButton:pressed
{
background-color:rgb(153, 147, 199);
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>230</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
font: 12pt &quot;Tw Cen MT Condensed Extra Bold&quot;;
background-color:rgb(179, 174, 218)
}

QPushButton:pressed
{
background-color:rgb(153, 147, 199);
}
</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>170</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
font: 12pt &quot;Tw Cen MT Condensed Extra Bold&quot;;
background-color:rgb(179, 174, 218)
}

QPushButton:pressed
{
background-color:rgb(153, 147, 199);
}
</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>170</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
font: 12pt &quot;Tw Cen MT Condensed Extra Bold&quot;;
background-color:rgb(179, 174, 218)
}

QPushButton:pressed
{
background-color:rgb(153, 147, 199);
}
</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>230</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
font: 12pt &quot;Tw Cen MT Condensed Extra Bold&quot;;
background-color:rgb(179, 174, 218)
}

QPushButton:pressed
{
background-color:rgb(153, 147, 199);
}
</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_next">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>350</y>
      <width>121</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
font: 12pt &quot;Tw Cen MT Condensed Extra Bold&quot;;
background-color:rgb(194, 179, 243)
}

QPushButton:pressed
{
background-color:rgb(153, 147, 199);
}</string>
    </property>
    <property name="text">
     <string>→</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>290</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
font: 12pt &quot;Tw Cen MT Condensed Extra Bold&quot;;
background-color:rgb(179, 174, 218)
}

QPushButton:pressed
{
background-color:rgb(153, 147, 199);
}
</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_equally">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>290</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
font: 30pt &quot;Tw Cen MT Condensed Extra Bold&quot;;
background-color:rgb(138, 132, 184)
}

QPushButton:pressed
{
background-color:rgb(153, 147, 199);
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_replace">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>230</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
font: 12pt &quot;Tw Cen MT Condensed Extra Bold&quot;;
background-color:rgb(194, 179, 243)
}

QPushButton:pressed
{
background-color:rgb(153, 147, 199);
}</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(226, 226, 255);
font: 12pt &quot;Tw Cen MT Condensed Extra Bold&quot;;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(226, 226, 255);
font: 12pt &quot;Tw Cen MT Condensed Extra Bold&quot;;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>50</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(226, 226, 255);
font: 12pt &quot;Tw Cen MT Condensed Extra Bold&quot;;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>50</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(226, 226, 255);
font: 12pt &quot;Tw Cen MT Condensed Extra Bold&quot;;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>50</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(226, 226, 255);
font: 12pt &quot;Tw Cen MT Condensed Extra Bold&quot;;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_show">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>110</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(226, 226, 255);
font: 12pt &quot;Tw Cen MT Condensed Extra Bold&quot;;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_befor">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(226, 226, 255);
font: 12pt &quot;Tw Cen MT Condensed Extra Bold&quot;;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_from">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(226, 226, 255);
font: 12pt &quot;Tw Cen MT Condensed Extra Bold&quot;;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clear">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>170</y>
      <width>120</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
font: 12pt &quot;Tw Cen MT Condensed Extra Bold&quot;;
background-color:rgb(194, 179, 243)
}

QPushButton:pressed
{
background-color:rgb(153, 147, 199);
}
</string>
    </property>
    <property name="text">
     <string>⌫</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clear_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>230</y>
      <width>120</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
font: 12pt &quot;Tw Cen MT Condensed Extra Bold&quot;;
background-color:rgb(194, 179, 243)
}

QPushButton:pressed
{
background-color:rgb(153, 147, 199);
}
</string>
    </property>
    <property name="text">
     <string>стереть всё</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Tw Cen MT Condensed Extra Bold&quot;;
</string>
    </property>
    <property name="text">
     <string>x³ +         x²+        x+         +</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 31pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>[   ;   ]</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;Berlin Sans FB Demi&quot;;</string>
    </property>
    <property name="text">
     <string>Калькулятор:  Метод половинного деления</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_accuracy">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>130</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(226, 226, 255);
font: 12pt &quot;Tw Cen MT Condensed Extra Bold&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>100</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Tw Cen MT Condensed Extra Bold&quot;;
</string>
    </property>
    <property name="text">
     <string>eps</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_root">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>170</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
font: 12pt &quot;Tw Cen MT Condensed Extra Bold&quot;;
background-color:rgb(194, 179, 243)
}

QPushButton:pressed
{
background-color:rgb(153, 147, 199);
}
</string>
    </property>
    <property name="text">
     <string>√</string>
    </property>
   </widget>
   <zorder>pushButton_e</zorder>
   <zorder>pushButton_0</zorder>
   <zorder>pushButton_7</zorder>
   <zorder>pushButton_4</zorder>
   <zorder>pushButton_1</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>pushButton_point</zorder>
   <zorder>pushButton_6</zorder>
   <zorder>pushButton_9</zorder>
   <zorder>pushButton_8</zorder>
   <zorder>pushButton_5</zorder>
   <zorder>pushButton_next</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>pushButton_equally</zorder>
   <zorder>pushButton_replace</zorder>
   <zorder>pushButton_clear</zorder>
   <zorder>pushButton_clear_2</zorder>
   <zorder>label</zorder>
   <zorder>lineEdit_3</zorder>
   <zorder>lineEdit_2</zorder>
   <zorder>lineEdit_1</zorder>
   <zorder>lineEdit_5</zorder>
   <zorder>lineEdit_4</zorder>
   <zorder>label_2</zorder>
   <zorder>lineEdit_befor</zorder>
   <zorder>lineEdit_from</zorder>
   <zorder>lineEdit_show</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>lineEdit_accuracy</zorder>
   <zorder>pushButton_root</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
